final_task/rss_reader/pars_args.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import sys
+import logging
+
+module_logger = logging.getLogger("rss_reader.scripts.pars_args")
+
+
+def create_parser():
+    """ function to parse the command line """
+    logger = logging.getLogger("rss_reader.scripts.create_parser")
+    logger.info("parse the command line ")
+    parser = argparse.ArgumentParser(
+        prog='rss_reader',
+        description=''' This program which receives RSS URL 
+                        and prints results in human-readable format.''',
+        epilog='''Thank you for using this program'''
+
+    )
+
+    # add information about the expected parameters
+    # using the add_argument method one call for each parameter).
+
+    parser.add_argument('source',
+                        type=str,
+                        default="not url",
+                        help='RSS URL')
+
+    parser.add_argument('--version',
+                        action='version',
+                        help='Print version info',
+                        version='%(prog)s {}'.format("1.1"))
+
+    parser.add_argument('--json',
+                        action='store_const',
+                        const=True,
+                        default=False,
+                        help='Print result as JSON in stdout')
+
+    parser.add_argument('--verbose',
+                        action='store_const',
+                        const=True,
+                        default=False,
+                        help='Outputs verbose status messages')
+
+    parser.add_argument('--limit',
+                        type=int,
+                        metavar='LIMIT',
+                        default=None,
+                        help='Limit news topics if this parameter provided')
+
+    return parser
```

Comment:
It is better to turn these into one-liners, they are pretty readable. Also it is better to use `formatted string` (`f"..."`) instead of `.format`.

Reply:
Done